scrape/scrape_test.go

```diff
@@ -484,8 +488,8 @@ func TestScrapePoolAppender(t *testing.T) {
 }
 
 func TestScrapePoolRaces(t *testing.T) {
-	interval, _ := model.ParseDuration("500ms")
-	timeout, _ := model.ParseDuration("1s")
+	interval, _ := model.ParseDuration("1s")
+	timeout, _ := model.ParseDuration("500ms")
```

Comment:
interval < timeout should be rejected

Reply:
But `1s` > `500ms`???

https://github.com/prometheus/prometheus/blob/ca526806485de720daa513b472bed68ea6c96033/scrape/target.go#L454-L456

Reply:
oops you are correct

mea culpa